src/snmp.c

```diff
@@ -1655,7 +1660,14 @@ static int csnmp_read_table(host_definition_t *host, data_definition_t *data) {
 
   status = 0;
   while (status == 0) {
-    req = snmp_pdu_create(SNMP_MSG_GETNEXT);
+    /* If SNMP v2 and later and bulk transfert enabled, use GETBULK PDU */
```

Comment:
maybe we should issue a warning here: `bulk transfer requested but unsupported for snmp version 1`

Reply:
I think, such should be done in config phase, not read.

Reply:
Added a warning in the config phase, i didn't wan't it to be blocking so it should not invalidate the host if i did not made a boo-boo.